js/dnt-helper.js

```diff
@@ -27,9 +27,9 @@ function _dntEnabled(dnt, userAgent) {
     // http://www.regexr.com/3c2el
     var platform = ua.match(/Windows.+?(?=;)/g);
 
-    // With old versions of IE, DNT did not exist so we simply return false;
+    // With old versions of IE, DNT did not exist so we assume it's disabled
     if (isIE && typeof Array.prototype.indexOf !== 'function') {
-        return false;
+        dntStatus = 'Disabled';
```

Comment:
@thefourtheye Thank you for your pull request. For some reason I never received a notification of this, and just came across this now. The reason I simply return `false` here instead of what you are suggesting, is that the code immediately stops executing here.

If we make the change you suggest here, the rest of the code below this will have to execute as well. And in the end it is still just going to return `false`.

Reply:
Sorry, I honestly forgot why I suggested this change. Obviously this is not needed. I'll close this PR now. Thanks for taking a look at it @schalkneethling

Reply:
Now worries, that for using the lib @thefourtheye